api/handlers.go

```diff
@@ -62,6 +62,59 @@ func getVersion(c *context, w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(version)
 }
 
+// GET /images/get
+func getImages(c *context, w http.ResponseWriter, r *http.Request) {
+	if err := r.ParseForm(); err != nil {
+		httpError(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	names := r.Form["names"]
+
+	// find an engine which has all images
+	// Engine.Addr : [found names]
+	dict := make(map[string][]string)
+	bFoundEngine := false
+	var foundEngineAddr string
+	for _, image := range c.cluster.Images() {
+		for _, name := range names {
+			if image.Match(name) {
+				// check if engine addr already exists
+				value, exists := dict[image.Engine.Addr]
+				if exists {
+					// check if name already exists
+					found := false
```

Comment:
Could you change this to something like `nameAlreadyExisted` instead of `found`?